src/org/intellij/erlang/debugger/xdebug/ErlangXDebugProcess.java

```diff
@@ -358,6 +361,25 @@ private OSProcessHandler runDebugTarget() throws ExecutionException {
     return erlangProcessHandler;
   }
 
+  private void setupErlangAppConfig(GeneralCommandLine commandLine) throws ExecutionException {
+    ErlangRunConfigurationBase<?> runConfig = getRunConfiguration();
+    if (runConfig.getDebugOptions().isLoadingConfig()) {
+      commandLine.addParameters("-config", runConfig.getDebugOptions().getAppConfig());
+    }
+  }
+
+  private void setupErlangRebarDependencies(GeneralCommandLine commandLine) throws ExecutionException {
+    ErlangRunConfigurationBase<?> runConfig = getRunConfiguration();
+    if (runConfig.getDebugOptions().isIncludingRebarDependencies()) {
+      Set<String> dependencies = RebarConfigUtil.getRebarDependencies(
+              runConfig.getConfigurationModule().getModule(),
+              runConfig.getDebugOptions().isFetchingDependencies());
+
+      for (String dep : dependencies)
+        commandLine.addParameters("-pa", dep);
+    }
```

Comment:
Regarding the import process, if this is intended to be the area which needs to be focused on for dependencies, I could probably drop the eunit config customizations for rebar deps and fix the issues here instead.